dev-manuals/architecture/chickenshock-backend/README.md

```diff
@@ -1,15 +1,32 @@
 # Architecture of Chickenshock-backend
 
-Project: <https://github.com/Gamify-IT/chickenshock-backend> \
-Classes: <https://github.com/Gamify-IT/chickenshock-backend/tree/main/src>
-
-## Purpose
-
-This backend saves relevant Chickenshock game data.
+Chickenshock is a minigame which is part of the Gamify-IT platform.
+The intention with this game is to make answering single choice questions fun.
+
+## Table of Contents
+
+* [Links](#links)
+* [General code structure](#general-code-structure)
+* [Starting the service](#starting-the-service)
+* [What to look at](#what-to-look-at)
+* [What to ignore](#what-to-ignore)
+* [Class diagrams](#class-diagrams)
+  * [Overview complete application](#overview-complete-application)
+  * [ConfigController](#configcontroller)
+  * [GameResultController](#gameresultcontroller)
+* [Known Design Flaws](#known-design-flaws)
+* [Known Bugs](#known-bugs)
+* [Tests](#tests)
+
+## Links
+
+- The [user docs](../../../user-manuals/minigames/chickenshock.md) explain how to play the game.
+- The program code is available at [here](https://github.com/Gamify-IT/chickenshock-backend).
+- The overview of the architecture can be found [here](../general-architecture.md).
 
 ## General code structure
 
-This backend has the following package structure
+This [Java code of the backend](https://github.com/Gamify-IT/finite-quiz-backend/blob/adef8dd1e681d655d26852d7c5741e254db16e57/src/main/java/de/unistuttgart/finitequizbackend) has the following package structure
```

Comment:
Why not the `main` links?
Also, that's a link to finitequiz in chickenshock documentation…

Reply:
```suggestion
This [Java backend](https://github.com/Gamify-IT/chickenshock-backend/tree/main/src/main/java/de/unistuttgart/chickenshockbackend) has the following package structure
```